src/WordPress/Proxy.php

```diff
@@ -102,7 +102,7 @@ public function createRequest()
      */
     public function getJSONBody()
     {
-        return file_get_contents('php://input');
+        return $GLOBALS[Hooks::CLOUDFLARE_JSON];
```

Comment:
This is not a straightforward thing to do. `getJSONBody()` and `getCloudflareRequestJSON()` needs docs explaining what this is. It's so hard for a new dev to grasp why we do something like this and it should be clear in the code with where it's being used and why.